app/controllers/admin/users_controller.rb

```diff
@@ -10,6 +10,19 @@ def index
       @users = User.all
     end
 
+    def toggle_confirmation
+      if params[:user][:should_be_confirmed] == 'true'
+        params[:user].delete(:should_be_confirmed)
+        params[:user][:confirmed_at] = Time.now
+        @user.update_attributes(user_params)
+      elsif params[:user][:should_be_confirmed] == 'false'
+        params[:user].delete(:should_be_confirmed)
```

Comment:
@differentreality `should_be_confirmed` is not a part of user model. Should a parameter not being a part of that model be whitelisted? If yes, I'll permit it in `user_params` too :).